sagemaker-keyspaces/README.md

```diff
@@ -0,0 +1,51 @@
+# Train Machine Learning Models using Amazon Keyspaces as a Data Source.  
+
+In this lab we will provide step-by-step instruction to use SageMaker to ingest customer data from Amazon Keyspaces and train a clustering model that allowed you to segment customers. You could use this information for targeted marketing, greatly improving your business KPI.
```

Comment:
For clarity, this is a "notebook", not a "lab".

Grammar nit: `model that allowed you` -> `model that allows you`